ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -1691,6 +1691,13 @@ odla_value odla_Softmax(odla_value input, odla_int32 axis,
   return v;
 }
 
+odla_value odla_LogSoftmax(odla_value input, odla_int32 axis,
+                           const odla_value_id id) {
```

Comment:
can you try https://docs.oneapi.io/versions/latest/onednn/group__dnnl__api__logsoftmax.html ?
It should have better performance than two-step computation.